packages/qwik/src/core/container/serializers.ts

```diff
@@ -442,6 +442,28 @@ const BigIntSerializer = /*#__PURE__*/ serializer<bigint>({
   },
 });
 
+const Uint8ArraySerializer = /*#__PURE__*/ serializer<Uint8Array>({
+  $prefix$: '\u001c',
+  $test$: (v) => v instanceof Uint8Array,
+  $serialize$: (v) => {
+    let buf = '';
+    for (const c of v) {
+      buf += String.fromCharCode(c);
+    }
+    return btoa(buf);
```

Comment:
You can make this a tiny bit smaller by doing `btoa(buf).replace(/=+$/, '')`

Reply:
Indeed. I fixed